### PR TITLE
12.0 fix reveal rar

### PR DESCRIPTION
**Pull request by rrahir**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/js/services/mail_notification_manager.js [hunk 1]
```diff
@@ -361,6 +361,8 @@ MailManager.include({
             this._handlePartnerMailFailureNotification(data);
         } else if (data.type === 'user_connection') {
             this._handlePartnerUserConnectionNotification(data);
+        } else if (data.type === 'simple_notification') {
+            data.warning ? this.do_warn(data.title, data.message, data.sticky) : this.do_notify(data.title, data.message, data.sticky);
```
Comment (alexkuhn) [inline addons/mail/static/src/js/services/mail_notification_manager.js]: `warning` not used yet. Could be introduced later when it's used.

Comment (rrahir) [reply addons/mail/static/src/js/services/mail_notification_manager.js]: I used in a previous version but I found it very agressive, I will run it with a PO to see if they want the warning or not for my usecase update the code if needed.

**Comment by rrahir** [comment]
@alexkuhn Can you have a quick look?

**Comment by rrahir** [comment]
@robodoo r+

**Comment by alexkuhn** [comment]
don't forget to squash fixup's ;)

**Comment by rrahir** [comment]
good call :p

**Comment by rrahir** [comment]
@robodoo  r+

**Pull request closed without merging**